components/collection/unlockable/CountdownTimer.vue

```diff
@@ -1,14 +1,15 @@
 <template>
-  <div ref="wrapper" class="px-4 py-4 keen-slider">
+  <div ref="wrapper" class="px-4 py-4 keen-slider background-inverse">
     <div
-      v-for="(section, index) in [1, 2, 3, 4]"
+      v-for="(section, index) in Array(slidesCountOnTimeCountdown).fill(0)"
       :key="index"
       class="keen-slider__slide is-flex no-wrap">
       <div class="no-wrap">
         <span class="mr-4 has-text-grey">Mint Phase Ends In</span
-        ><span
-          >{{ hours }} Hours : {{ minutes }} Minutes :
-          {{ seconds }} Seconds</span
+        ><span class="has-text-inverse"
+          >{{ getPositiveNumber(hours) }} Hours :
+          {{ getPositiveNumber(minutes) }} Minutes :
+          {{ getPositiveNumber(seconds) }} Seconds</span
```

Comment:
```suggestion
          >{{ getPositiveNumber(hours) }} Hours:
          {{ getPositiveNumber(minutes) }} Minutes:
          {{ getPositiveNumber(seconds) }} Seconds</span
```

Reply:
I think there is no need to change the space between the text.
<img width="363" alt="image" src="https://github.com/kodadot/nft-gallery/assets/31397967/cb649b9e-1022-42d9-8830-b5e981b693ad">